src/controllers/librarysettings.js

```diff
@@ -1,48 +1,46 @@
-define(["jQuery", "loading", "libraryMenu", "fnchecked", "emby-checkbox", "emby-linkbutton"], function($, loading, libraryMenu) {
+define(["jQuery", "loading", "libraryMenu", "fnchecked", "emby-checkbox", "emby-button"], function($, loading, libraryMenu) {
     "use strict";
 
     function loadPage(page, config) {
-        $("#chkSaveMetadataHidden", page).checked(config.SaveMetadataHidden);
-        loading.hide();
+        config.MergeMetadataAndImagesByName ? $(".fldImagesByName", page).hide() : $(".fldImagesByName", page).show(), $("#chkSaveMetadataHidden", page).checked(config.SaveMetadataHidden), $("#txtMetadataPath", page).val(config.MetadataPath || ""), $("#txtMetadataNetworkPath", page).val(config.MetadataNetworkPath || ""), loading.hide()
```

Comment:
Several files have been reverted to a previous state actually.

Reply:
I restored to master version, and then i proceeded in replacing emby-linkbutton to emby-button.